app/scripts/services/environment.js

```diff
@@ -8,6 +8,11 @@ angular.module("openshiftConsole")
       if (apiObject.kind === 'Pod') {
         return _.get(apiObject, 'spec.containers', []);
       }
+      // TODO: this isn't actually containers....
```

Comment:
So this just gets the component to display.  However, its incorrect.

If this component is the way to go, I'd prob go back up into the `editEnvironmentVariables` component & test for `BuildConfig`, then alter the display/template.  Looking into this now.